terra_layer/views/layers.py

```diff
@@ -26,7 +26,7 @@ class SceneViewset(ModelViewSet):
     permission_classes = (ScenePermission,)
 
     def get_serializer_class(self,):
-        if self.action == "retrieve":
+        if self.action in ["retrieve", "update", "create"]:
```

Comment:
Why authorize these other actions?

Reply:
We want the detail serializer with the optionnal slug field for this to action.